decide/src/main/java/org/example/Decide.java

```diff
@@ -55,7 +55,9 @@ public boolean condition1() {
     }
 
     public boolean condition2() {
-        return true;
+        return settings.NUMPOINTS >= 3 && IntStream.range(0, settings.NUMPOINTS - 2).anyMatch(
+                (index) -> (settings.POINTS[index + 1].angle(settings.POINTS[index], settings.POINTS[index + 2])
+                        > Math.PI - settings.PARAMETERS.EPSILON));
```

Comment:
I think we're missing the other case here, it's either:
angle > PI + EPSILON
or
angle < PI - EPSILON

Reply:
Thanks for the catch!